.github/workflows/ddl-changes.yml

```diff
@@ -20,13 +20,19 @@ jobs:
         with:
           clean: false
           fetch-depth: 200
-      - uses: actions/setup-python@v5
+
+      - uses: astral-sh/setup-uv@884ad927a57e558e7a70b92f2bccf9198a4be546 # v6
         with:
-          python-version: '3.11'
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          make install-python-dependencies
+          version: '0.8.2'
+          # we just cache the venv-dir directly in action-setup-venv
+          enable-cache: false
+
+      - uses: getsentry/action-setup-venv@3a832a9604b3e1a4202ae559248f26867b467cc7 # v2.1.1
+        with:
+          python-version: 3.11.11
```

Comment:
Can we read this value from `.python-version` so this stays in sync with that?

Reply:
https://github.com/getsentry/action-setup-venv/pull/22